### ISSUE TITLE
Broken links in video tabs

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-to-hugo#481]
In https://github.com/mitodl/ocw-data-parser/pull/165 we added overlooked metadata that is included on video resources. This metadata often includes links to PDFs. 

### Steps to Reproduce

1. Visit, for example, https://ocwnext.odl.mit.edu/courses/res-18-005-highlights-of-calculus-spring-2010/resources/derivative-of-sin-x-and-cos-x/ 
2. Click the `+` to reveal the related resources 
3. Click the PDF link

### Expected Behavior

- Ideally, you would be linked to a resource page with an embedded PDF
- failing that, you should at least download the PDF

### Actual Behavior

- S3 `NoSuchKey` error

**Comment by pdpinch** [comment]
@mbertrand are these PDFs getting imported?

@ChristopherChudzicki are these links on your radar for broken link fixes?

**Comment by mbertrand** [comment]
The pdf file is getting imported, here is the direct url to it: https://ocwnext-rc.odl.mit.edu/courses/res-18-005-highlights-of-calculus-spring-2010/618734d60a968b2b1cb2d8c8f0a80ef0_MITRES18_05S10_Der_sin_x_and_cos_x.pdf

The resource: https://ocw-studio.odl.mit.edu/admin/websites/websitecontent/468401/change/
In github: 
https://github.mit.edu/mitocwcontent/res.18-005-spring-2010/blob/main/content/resources/mitres18_05s10_der_sin_x_and_cos_x.md

FWIW I'm not seeing any markdown when going to the edit resource url for that video:
https://ocw-studio.odl.mit.edu/sites/res-18-005-highlights-of-calculus-spring-2010/type/resource/edit/7eb3ca6c-874d-219f-37c6-afd983dc2bb3/

**Comment by ChristopherChudzicki** [comment]
No, these have no been on my radar. The `markdown_cleanup` management command has only been targeting markdown bodies, not metadata.

I think it should be relatively simple to reuse the URL-fixing code for metadata, just need to double check whether we can use shortcodes in the metadata or not.

**Comment by ChristopherChudzicki** [comment]
I believe the issue is just that the related resource links are relative
```yaml
related_resources_text: Lecture summary and Practice problems ([PDF](resources/mitres18_05s10_der_sin_x_and_cos_x
  "Open in a new window."))
```
Changing `(resources/...)` to `(/resources/...)` should do it.

**Comment by ChristopherChudzicki** [comment]
Closing this. The fix has been run on prod; a few links are still broken because they are `resolveuid`-based, but that's tracked in https://github.com/mitodl/ocw-studio/issues/1116
